server/embed/etcd.go

```diff
@@ -790,7 +790,7 @@ func (e *Etcd) grpcGatewayDial(splitHttp bool) (grpcDial func(ctx context.Contex
 	addr := sctx.addr
 	if network := sctx.network; network == "unix" {
 		// explicitly define unix network for gRPC socket support
-		addr = fmt.Sprintf("%s://%s", network, addr)
```

Comment:
is this indicative of changes / breaks that will happen for etcd clients if this update merges?

Reply:
No, it doesn't impact K8s at all; it doesn't impact any etcd clients at all. It just changes the way how the grpc-gateway connects to the backend etcdserver. The change is transparent to any clients.

Reply:
> Connecting to unix:localhost:port instead of never officially supported unix://localhost:port (https://grpc.github.io/grpc/cpp/md_doc_naming.html)

Reply:
3.5 server does not allow client dialing with UDS. See https://github.com/etcd-io/etcd/pull/15940